ros2_graph/rosElement.py

```diff
@@ -0,0 +1,169 @@
+#! /usr/bin/env python3
+# Copyright 2023 Kiwicampus Inc.
+#
+# Licensed under GNU GENERAL PUBLIC, version 3 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.gnu.org/licenses/gpl-3.0.en.html
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from dataclasses import dataclass
+from enum import Enum
+from typing import List, Dict
+
+
+class ELEMENT_TYPE(Enum):
```

Comment:
Normally Enum class should follow CamelCase naming, not all as UPPERCASE since this could be misleading. UPPERCASE is normally used for constants

Reply:
Done